otx/algorithms/detection/configs/instance_segmentation/resnet50_maskrcnn/tile_pipeline.py

```diff
@@ -54,7 +54,7 @@
 __dataset_type = "CocoDataset"
 __data_root = "data/coco/"
 
-__samples_per_gpu = 4
+__samples_per_gpu = 12
```

Comment:
I think we can leave as 4

Reply:
https://github.com/openvinotoolkit/training_extensions/pull/2172/commits/3cdd32725a5155c9e5f304c6cb372a87bafee917